ams/server/src/main/java/com/netease/arctic/server/ArcticServiceContainer.java

```diff
@@ -473,6 +475,16 @@ private void validateThreadCount(
       }
     }
 
+    /** Override the value of {@link SystemProperties}. */
+    private void setIcebergSystemProperties() {
+      int workerThreadPoolSize =
+          Math.max(
+              Runtime.getRuntime().availableProcessors(),
+              serviceConfig.getInteger(ArcticManagementConf.TABLE_MANIFEST_IO_THREAD_COUNT));
+      System.setProperty(
+          SystemProperties.WORKER_THREAD_POOL_SIZE_PROP, String.valueOf(workerThreadPoolSize));
```

Comment:
I think we should avoid directly setting system variables. There are many different plan scenarios now, and we should differentiate their thread pools. How do you think?@Zhoujinsong